openpype/hosts/houdini/plugins/publish/extract_fbx.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""Fbx Extractor for houdini. """
+
+import os
+import pyblish.api
+from openpype.pipeline import publish
+from openpype.hosts.houdini.api.lib import render_rop
+
+import hou
+
+
+class ExtractFBX(publish.Extractor):
+
+    label = "Extract FBX"
+    families = ["fbx"]
+    hosts = ["houdini"]
+
+    order = pyblish.api.ExtractorOrder + 0.1
+
+    def process(self, instance):
+
+        # get rop node
+        ropnode = hou.node(instance.data.get("instance_node"))
+
+        # render rop
+        render_rop(ropnode)
+
+        # get required data
+        file_name, staging_dir = self.get_paths_data(ropnode)
+        representation = self.get_representation(instance,
```

Comment:
`get_representation` is a confusing name since it doesn't get or query an existing entry. I'd rather call it `prepare_representation`